custom_components/opnsense/coordinator.py

```
@@ -70,12 +92,240 @@ async def _get_states(self, categories: list) -> MutableMapping[str, Any]:
+                            if "pkts" in prop_name or "bytes" in prop_name:
+                                (
+                                    new_property,
+                                    value,
+                                ) = await OPNsenseDataUpdateCoordinator._calculate_speed(
+                                    prop_name=prop_name,
+                                    elapsed_time=elapsed_time,
+                                    current_parent_value=instance[prop_name],
+                                    previous_parent_value=previous_instance[prop_name],
+                                )
+
+                            instance[new_property] = value
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)